packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-## [2.0.0] - TBD
+## [2.1.0] - TBD
+* Bug fixing
```

Comment:
Add information for the dashboard feature and the new param --enable-feature-counter

Reply:
Done